model/task/task.go

```diff
@@ -565,6 +566,13 @@ func (t *Task) MarkEnd(finishTime time.Time, detail *apimodels.TaskEndDetail) er
 		if timedOutStart.Before(t.CreateTime) {
 			t.StartTime = t.CreateTime
 		}
+		grip.Warning(message.Fields{
+			"message":      "Task is missing start time",
+			"task_id":      t.Id,
```

Comment:
Instead of logging just the ID, it might make sense to log the entire document here (assuming this happens very rarely)